.circleci/config.yml

```diff
@@ -19,6 +19,68 @@ jobs:
             cargo install cargo-audit
       - run:
           command: cargo audit
+
+  test:
+    docker:
+      - image: circleci/python:2.7
+      - image: circleci/dynamodb
+        command: -sharedDb -inMemory
+    environment:
+      AWS_LOCAL_DYNAMODB: http://localhost:8000
+    steps:
+      - checkout
+      # Need to download the requirements.txt files so we can use their
+      # checksums in restore_cache.
+      - run:
+          name: Download Python dependency lists
+          command: |
+            curl https://raw.githubusercontent.com/mozilla-services/autopush/master/requirements.txt > requirements.txt
+            curl https://raw.githubusercontent.com/mozilla-services/autopush/master/test-requirements.txt > test-requirements.txt
+      - restore_cache:
+          name: Restoring Python cache
+          key: python-v1-{{ checksum "requirements.txt" }}-{{ checksum "test-requirements.txt" }}
+      - restore_cache:
+          name: Restoring Rust cache
+          key: rust-v1-{{ .Branch }}-{{ checksum "Cargo.lock" }}
+      - run:
+          name: Set up Python
+          command: |
+            pip install --upgrade pip
+            pip install bottle
+            pip install -r requirements.txt
+            pip install -r test-requirements.txt
```

Comment:
IIRC, `test-requirements.txt` begins with a `-r requirements.txt`. You should be able to just bring it in and it'll grab both files.